tests/test_codegen.py

```diff
@@ -1285,6 +1286,41 @@ class Simple(Base):
 """
 
 
+@pytest.mark.skipif(sqlalchemy.__version__ < '1.2', reason='Requires SQLAlchemy 1.2+')
+def test_table_comment(metadata):
+    Table(
+        'simple', metadata,
+        Column('id', INTEGER, primary_key=True),
+        comment="this is a 'comment'"
+    )
+
+    codegen = CodeGenerator(metadata)
+    code = codegen.render_table(codegen.models[0])
+    expected = dedent("""
```

Comment:
Why use dedent here specifically and not elsewhere?

Reply:
No particularly good reason - I can revert that.